src/components/BookElement/BookElement.jsx

```diff
@@ -0,0 +1,50 @@
+
+import React, {forwardRef} from "react";
+import {CardActionArea, Card, CardContent, CardMedia, Typography, Grid} from "@material-ui/core";
+import MoreVertIcon from "@material-ui/icons/MoreVert";
+import {useBookStyle} from './BookElement.style'
+
+export const BookElement = forwardRef((props,ref) => {

```

Comment:
no need for Element or Components sincew we are in components anyway  so Book is enough